src/anomalib/cli/cli.py

```diff
@@ -13,6 +13,7 @@
 
 from jsonargparse import ActionConfigFile, ArgumentParser, Namespace
 from jsonargparse._actions import _ActionSubCommands
+from lightning.pytorch.core.datamodule import LightningDataModule
```

Comment:
After https://github.com/openvinotoolkit/anomalib/pull/1748, we may not have lightning installed initially. Does this consider this?

Reply:
Ah good point! I have changed the import location